app/Http/Resources/ScheduleDetailedResource.php

```
@@ -0,0 +1,41 @@
+            "scheduleType" => $this->scheduleType,
+            "daysOfWeek" => $this->daysOfWeek,
+            "startRecur" => $this->startRecur?->format('Y-m-d'),
+            "endRecur" => $this->endRecur?->format('Y-m-d'),
+            "startTime" => $this->startTime?->format('H:i'),
+            "endTime" => $this->endTime?->format('H:i'),
+            "day_of_week_names_short" => $this->day_of_week_names_short,
+            "day_of_week_names" => $this->day_of_week_names,
+            "created_at" => $this->created_at,
+            "updated_at" => $this->updated_at,
+            "deleted_at" => $this->deleted_at,
+            "department" => SyncListDepartmentResource::make($this->whenLoaded('department')),
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\SyncListDepartmentResource' in method 'toArray'. (Clean Code Rules)

(StaticAccess)